### PR TITLE
moving random.go from utils

### PR DESCRIPTION
**Pull request by brahmaroutu**
Addresses #10962
Signed-off-by: Srini Brahmaroutu srbrahma@us.ibm.com

### CODE REVIEW

**Comment by brahmaroutu** [comment]
@crosbymichael I have moved random into pkg/common as I feel the code belongs there. Please let me know if you would want to refactor differently.

File: pkg/common/randomid_test.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+func TestShortenIdInvalid(t *testing.T) {
+	id := "1234"
+	truncID := TruncateID(id)
+	if len(truncID) != len(id) {
+		t.Fatalf("Id returned is incorrect: truncate on %s returned %s", id, truncID)
+	}
+}
+
+func TestGenerateID(t *testing.T) {
```
Comment (ahmetalpbalkan) [inline pkg/common/randomid_test.go]: should be `TestGenerateRandomID`

**Comment by ahmetalpbalkan** [comment]
This package is turning into a `randomutils` and probably should initialize its own instance of `rand.Random` in an `init` function seeded with time. Otherwise it's going to generate same IDs over and over again. Also it's better if this does not rely on the pkg `math/rand`'s globalRandom instance.

File: pkg/common/randomid_test.go (hunk 1 above)
Comment (brahmaroutu) [inline pkg/common/randomid_test.go]: @ahmetalpbalkan Fixed.

**Comment by brahmaroutu** [comment]
I am not sure if we should address the concern here. Please keep the scope of this to moving out of Utils. We can open a new issue on how Random generation of IDs work.

**Comment by ahmetalpbalkan** [comment]
@brahmaroutu You're right it's out of scope for this PR.

LGTM

**Comment by estesp** [comment]
LGTM

**Pull request merged**